ci-operator/step-registry/cucushift/e2e/cucushift-e2e-commands.sh

```diff
@@ -74,10 +74,19 @@ function filter_test_by_sno() {
     fi
     echo_e2e_tags
 }
+function filter_test_by_fips() {
+    local fips
+    fips="$(oc get configmap cluster-config-v1 -n kube-system -o yaml | yq '.data' | grep -i 'fips')"
```

Comment:
What would be the exact value returned from this one-liner?

Just in case an error text returned and it entered the if condition next to this line.

Reply:
If a cluster enabled FIPS, then we will get `fips: true`.  Or else we are supposed to get null

Reply:
Update the logic in https://github.com/openshift/release/compare/70a4386da07085207be2c62087e4cdc93958ca9f..f90cef8f4c9fa7a66543aec9a509b5ae932d5c50, as the `grep -i 'fips'` returned non-zero exit status and break the script on non-fips cluster.

Reply:
Thank you